lib/infrastructure/database-stats-repository.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
     return scalingoApi.getDbConnectionString(scalingoApp);
   },
 
-  async getQueryStats(scalingoApi, scalingoApp) {
+  async getQueryStats(scalingoApp) {
```

Comment:
Ah maintenant qu'on a des tests, on va pouvoir refacto sans :cold_sweat: 
Je propose de corriger un "bug" en mettant en clair que `stat` est `statistics` et pas `statements`
```suggestion
  async getQueryStatistics(scalingoApp) {
```
Il y a surtout les deux variables d'env qui s'appellent aujourd'hui
```
FT_STATEMENTS
STATEMENTS_SCHEDULE
```

Reply:
Je pense dans ce cas que quelque chose comme `getQueryStatStatements` serait plus correct.

L'action lancée dans scalingo est : `pg-stat-statements-list`, donc la notion de statements est importante aussi.

Reply:
Je pensais à ce que les `runningQueries` ne soient pas confondues avec les `statistics`

Le terme de "statements" ne m'est pas familier, à priori dans la doc PG
- une query c'est un client qui envoie du texte SQL à un serveur 
- un statement c'est le texte SQL

Donc bon, c'est intéressant de savoir la différence, mais `getQueryStatStatements` est mystérieux.
Je propose `getQueryStatistics` ou au pire `getQueryStatementsStatistics`

>The pg_stat_statements module provides a means for tracking planning and execution statistics of all SQL statements executed by a server.
https://www.postgresql.org/docs/current/pgstatstatements.html

Reply:
Ah oui il ne faut pas non plus confondre avec les stats de serveur :thinking:

Reply:
Bon allez on merge tel quel